catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/RemoveCommand.java

```diff
@@ -81,53 +90,49 @@ private Object executeRemoveFromCache() throws Exception {
     }
 
     private Object executeRemoveFromStore() throws Exception {
+
         CatalogFacade catalogProvider = getCatalog();
 
+        int batchCount = 1;
+        int deletedCount = 0;
+
         if (hasFilter()) {
             QueryImpl query = new QueryImpl(getFilter());
```

Comment:
@emanns95 @lessarderic Sorry I missed the convo but I don't think we want the default timeout... because the default is infinite.

Reply:
I thought we updated the default to be 5 minutes?
https://codice.atlassian.net/browse/DDF-2862